documents/src/pages/elements/datetime-picker.md

```diff
@@ -573,18 +573,18 @@ datetimePicker?.addEventListener('before-cell-render', (event) => {
 ```
 ```css
 html[prefers-color-scheme="light"] {
-  --custom-cell-background-color: white;
+  --custom-cell-background-color: #ffffff00;
   --custom-cell-selected-background-color: rgb(51, 75, 255);
   --custom-cell-hover-color: rgb(20, 41, 189);
-  --custom-cell-range-background-color: rgba(51, 75, 255, 0.2);  
+  --custom-cell-range-background-color: rgba(51, 75, 255, 0.2);
   --custom-cell-highlight-color: #F5475B;
 }
 
 html[prefers-color-scheme="dark"] {
-  --custom-cell-background-color: rgb(13, 13, 13);
+  --custom-cell-background-color: #0d0d0d00;
   --custom-cell-selected-background-color: rgb(51, 75, 255);
   --custom-cell-hover-color: rgb(20, 41, 189);
-  --custom-cell-range-background-color: rgba(51, 75, 255, 0.2);  
+  --custom-cell-range-background-color: rgba(51, 75, 255, 0.2);
```

Comment:
You should use the only one way to set color with alpha between the Hex or the rgba for consistency